### ISSUE TITLE
aria-describedby not removed

### ISSUE THREAD

**Issue by bluerouse** [jquery-validation/jquery-validation#1299]
I just tested Release 1.13.1 and it seems, the issue with the duplicates-issues from #1200 are solved. But what I observed is, that the describedby is not resetted or removed, when errormessage has been cleared. So for my case, aria-describedby is added to link the errormessage, but it remains there forever, even if field is valid afterwards and the related id even does not exist anymore.
This is wrong, as the aria-attributes then points to a non-existing id, which is probably not accessible.

**Comment by rjgotten** [comment]
@jzaefferer 

> As long as the referenced element is empty or hidden, the effect is the same as removing the attribute.

It's not the same effect if the element is hidden. Elements referenced via `aria-labelledby` or `aria-describedby` are sourced for label or description whether they are visible or not.

https://www.paciellogroup.com/blog/2015/05/short-note-on-aria-labelledby-and-aria-describedby/

It's only the same effect when they are emptied.

**Comment by staabm** [comment]
@rjgotten anything on our end we need to change? If so please open a new issue and add some more background
